common/lib/xmodule/xmodule/modulestore/draft_and_published.py

```diff
@@ -167,8 +171,8 @@ def update_item_parent(self, item_location, new_parent_location, old_parent_loca
             self.update_item(old_parent_item, user_id)  # pylint: disable=no-member
             log.info(
                 '%s removed from %s children',
-                unicode(source_item.location),
-                unicode(old_parent_item.location)
+                six.text_type(source_item.location),
+                six.text_type(old_parent_item.location)
```

Comment:
`text_type` was already being imported directly, modernize just didn't notice.  The `six.` prefix can be omitted from these (same for the changes below in this file).

Reply:
Don't merge for now, I'll will correct that and keep you updated

Reply:
I've removed prefix, @jmbowman we are good to go :)